Sources/Valet/Internal/SecItem.swift

```diff
@@ -127,46 +92,36 @@ internal final class SecItem {
         
         if status == errSecSuccess {
             if let result = result as? DesiredType {
-                return .success(result)
+                return result
                 
             } else {
                 // The query failed to pull out a value object of the desired type, but did find metadata matching this query.
                 // This can happen because either the query didn't ask for return data via [kSecReturnData : true], or because a metadata-only item existed in the keychain.
-                return .error(errSecItemNotFound)
+                throw KeychainError.itemNotFound
             }
             
         } else {
-            ErrorHandler.assert(status != errSecMissingEntitlement, "A 'Missing Entitlements' error occurred. This is likely due to an Apple Keychain bug. As a workaround try running on a device that is not attached to a debugger.\n\nMore information: https://forums.developer.apple.com/thread/4743")
-            
-            return .error(status)
+            throw KeychainError(status: status)
         }
     }
     
-    internal static func containsObject(matching query: [String : AnyHashable]) -> Result {
+    internal static func containsObject(matching query: [String : AnyHashable]) -> OSStatus {
         guard query.count > 0 else {
```

Comment:
nit: [Let's use `isEmpty`](https://www.hackingwithswift.com/articles/181/why-using-isempty-is-faster-than-checking-count-0)

Reply:
addressed in 249188fdd7f7d62bd337d6067ed0bd051c66d5a6